### PR TITLE
Benchmarks - Fix missing include in FP8 benchmark

### PR DESCRIPTION
**Pull request by rhewett**
**Description**

I couldn't build the fp8 benchmark.

`
[ 25%] Building CXX object CMakeFiles/cublaslt_utils.dir/cublaslt_utils.cc.o
In file included from <scrubbed>/superbench/benchmarks/micro_benchmarks/cublaslt_fp8_gemm/cublaslt_utils.cc:4:
<scrubbed>/superbench/benchmarks/micro_benchmarks/cublaslt_fp8_gemm/cublaslt_utils.h: In function ‘void checkCublasStatus(cublasStatus_t)’:
<scrubbed>/superbench/benchmarks/micro_benchmarks/cublaslt_fp8_gemm/cublaslt_utils.h:15:20: error: ‘logic_error’ is not a member of ‘std’
   15 |         throw std::logic_error("cuBLAS API failed");
      |                    ^~~~~~~~~~~
<scrubbed>/superbench/benchmarks/micro_benchmarks/cublaslt_fp8_gemm/cublaslt_utils.cc: In member function ‘size_t cublasLtGemm::GetAlgorithm(int, size_t)’:
<scrubbed>/superbench/benchmarks/micro_benchmarks/cublaslt_fp8_gemm/cublaslt_utils.cc:103:20: error: ‘runtime_error’ is not a member of ‘std’
  103 |         throw std::runtime_error("Unable to find any suitable algorithms");
      |                    ^~~~~~~~~~~~~
make[2]: *** [CMakeFiles/cublaslt_utils.dir/build.make:76: CMakeFiles/cublaslt_utils.dir/cublaslt_utils.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:85: CMakeFiles/cublaslt_utils.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
`

Adding stdexcept fixed this.

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

**Review by abuccts — APPROVED** [review]
what's your gcc version by the way, >10?

File: superbench/benchmarks/micro_benchmarks/cublaslt_fp8_gemm/cublaslt_utils.h [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 #include <memory>
 #include <stdio.h>
+#include <stdexcept>
```
Comment (abuccts) [inline superbench/benchmarks/micro_benchmarks/cublaslt_fp8_gemm/cublaslt_utils.h:8]: pls sort includes (put it before stdio) to pass the clang lint

**Review by yukirora — APPROVED** [review]

**Comment by rhewett** [comment]
> what's your gcc version by the way, >10?

Im using GCC 11.3.0

**Pull request merged**